core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/bigquery/ServiceBigqueryDatabaseConnection.java

```diff
@@ -0,0 +1,38 @@
+package io.metadew.iesi.connection.database.bigquery;
+
+import io.metadew.iesi.connection.database.bigquery.BigqueryDatabaseConnection;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class ServiceBigqueryDatabaseConnection extends BigqueryDatabaseConnection {
+
+    public ServiceBigqueryDatabaseConnection(String hostName, int portNumber, String project, String serviceAccount, String keyPath) {
+        super(getConnectionUrl(hostName, portNumber, project, serviceAccount, keyPath));
+    }
+
+    private static String getConnectionUrl(String hostName, int portNumber, String project, String serviceAccount, String keyPath) {
+        StringBuilder connectionUrl = new StringBuilder();
+        connectionUrl.append("jdbc:bigquery://");
+        connectionUrl.append(hostName);
+        if (portNumber > 0) {
+            connectionUrl.append(":");
+            connectionUrl.append(portNumber);
+        }
```

Comment:
port is listed as a mandatory parameter in 'BigqueryDatabaseService'. When would the port be smaller than zero?